core/src/trezor/ui/layouts/progress.py

```diff
@@ -40,7 +40,7 @@ def __init__(
             ui.refresh()
         ui.backlight_fade(ui.BacklightLevels.NORMAL)
 
-    def set_timer(self, token: int, deadline: int) -> None:
+    def set_timer(self, token: int, duration: int) -> None:
```

Comment:
This was the problem also before but I don't like that the unit of `duration` is not apparent until very deep in the code (instantiation of `Duration` in rust). Can we either document that `duration` is in milliseconds or rename to `duration_ms`?

Reply:
like so? https://github.com/trezor/trezor-firmware/pull/4180/commits/13dd0bad8637b748dd781fc8f4a9c5152b3e8958